5.application_scopes.md

```diff
@@ -1,4 +1,5 @@
-# 4. Application Scopes
+# 5. Application Scopes
+**Target User: Application Operator**
```

Comment:
Aren't both Scope and Trait defined by Infra Ops? They define the spec, and app ops use it.

Reply:
It's another reason I begin to prefer "Target Audiences", both Infra Ops and App Ops are audiences for this doc in my mind. In the future we could cut a clearer boundary.